### PR TITLE
[css-view-transitions-2] Updates for horizontal review

### PR DESCRIPTION
**Pull request by khushalsagar**
Applies suggestions from https://github.com/w3c/csswg-drafts/issues/10283.

### CODE REVIEW

File: css-view-transitions-2/Overview.bs [hunk 1]
```diff
@@ -132,7 +132,7 @@ In cross-document view transition, what triggers a view transition is a navigati
 
 	* Both documents are of the [=same origin=];
 	* The page is visible throughout the entire course of the navigatiion;
```
Comment (vmpstr) [inline css-view-transitions-2/Overview.bs:134]:
```suggestion
	* The page is visible throughout the entire course of the navigation;
```

File: css-view-transitions-2/Overview.bs [hunk 2]
```diff
@@ -132,7 +132,7 @@ In cross-document view transition, what triggers a view transition is a navigati
...
-	* the navigation is initiated by the page, e.g. by clicking a link or submitting a form, or is a {{NavigationType/traverse}} navigation (back/forward). This excludes, for example, navigations initiated by the URL bar;
+	* The user initiates the navigation by interacting with the page, e.g. by clicking a link or submitting a form; or by by interacting with the browser UI to do a {{NavigationType/traverse}} navigation (back/forward). This excludes, for example, navigations initiated by the URL bar;
```
Comment (vmpstr) [inline css-view-transitions-2/Overview.bs:135]:
```suggestion
	* The user initiates the navigation by interacting with the page, e.g. by clicking a link or submitting a form; or by interacting with the browser UI to do a {{NavigationType/traverse}} navigation (back/forward). This excludes, for example, navigations initiated by the URL bar;
```

Comment (pes10k) [inline css-view-transitions-2/Overview.bs:135]: you've got a new typo here (by by) :)

**Review by vmpstr — APPROVED** [review]

File: css-view-transitions-2/Overview.bs (hunk 2 above)
Comment (khushalsagar) [reply css-view-transitions-2/Overview.bs]: Done!

**Pull request merged**